### ISSUE TITLE
Gridview not working

### ISSUE THREAD

**Issue by sajalhome** [Fugiman/google-meet-grid-view#265]
Since July the extension stopped working all of a sudden. It was working great till then. Now when the gridview is activated, I'm only able to see my video and all the participants are obscured. Checked the comments on github and chrome store, other users are also facing the same issue. Request you to please fix this great extension

**Comment by TitanessOfMind** [comment]
I Agree gridview has stopped working. When will this issue be fixed?  Need it for online learning. :(

**Comment by BrownComoSLP** [comment]
My grid view no longer includes me in the grid or when I present my screen.  My school district has vetted google meet as the primary tele therapy communication with families.  After deleting and reinstalling, updating google Chrome the google grid view still does not work as of 7/1/20.  Any help would be appreciated. Thanks

**Comment by icysapphire** [comment]
@TitanessOfMind @BrownComoSLP  @sajalhome 
https://chrome.google.com/webstore/detail/google-meet-grid-view-fix/dakebdbeofhmlnmjlmhjdmmjmfohiicn
This is just a temporary workaround (which looses the pinning feature) [credits: @ma2shita], in order to allow people who heavily rely on the extension (e.g., online exams) to continue their work. The extension will be removed as soon as Chris is able to properly fix the issue. Just temporary disable the problematic extension and activate this one.

**Comment by sajalhome** [comment]
That's a life saver!! Thanks a lot

**Comment by TitanessOfMind** [comment]
Thank you so much! We will have a trial class this week. Indeed a life
saver! 😊
